apps/src/templates/instructions.html.ejs

```diff
@@ -1,9 +1,9 @@
-<% var msg = require('../locale'); %>
-
-<p class='dialog-title'><%= msg.puzzleTitle(locals) %></p>
-<% if (locals.instructions) { -%>
+<p class='dialog-title'><%= locals.puzzleTitle %></p>
+<% if (locals.renderedMarkdown) { -%>
+  <%- locals.renderedMarkdown %>
```

Comment:
[Oddly enough](http://stackoverflow.com/questions/10326950/render-a-variable-as-html-in-ejs/10330401#10330401), <%- is how to render something un-escaped